integration-tests/src/test/java/uk/gov/di/authentication/api/AuthExternalApiUserInfoIntegrationTest.java

```diff
@@ -294,6 +292,7 @@ private void withAuthSessionNewAccount() {
                         .getSession(TEST_SESSION_ID)
                         .get()
                         .withAccountState(AuthSessionItem.AccountState.NEW)
-                        .withVerifiedMfaMethodType(MFAMethodType.AUTH_APP.getValue()));
+                        .withVerifiedMfaMethodType(MFAMethodType.AUTH_APP.getValue())
+                        .withCurrentCredentialStrength(CredentialTrustLevel.MEDIUM_LEVEL));
```

Comment:
This also doesn't seem to be being tested. Also, question: if this is a NewAccount, then would their currentCredentialStrength always be null at this point?

Reply:
Same as above where following the pattern of including an example but don't know if the claim needs to be specifically tested here.

Reply:
If its a new account it would almost always be non-null as for NewAccount i believe verification always goes through the verifyMfaCode lambda which sets the current credential value.

Reply:
I think I agree here, that nothing new is being tested. I personally would read this line as saying, "it's important for the authSession to have this property for this test". Certainly if in 6 months I go to uses of the `withCurrentCredentialStrength` method. So I'd either remove it, or add an assertion. Happy with which ever you decide on this one:)

Reply:
Cool I'll make one of those changes :)